gio/Gir.toml

```diff
@@ -546,6 +546,8 @@ manual_traits = ["FileExtManual"]
         [[object.function.parameter]]
         name = "arg"
         string_type = "os_string"
+        [object.function.return]
+        nullable = true
```

Comment:
This can never be nullable per the docs "This operation never fails, but the returned object might not support any I/O operation if arg points to a malformed path."

Reply:
The issue seems to say otherwise though...

Reply:
And the source code seems to do it too: https://gitlab.gnome.org/GNOME/glib/-/blob/main/gio/gfile.c#L6728

The doc is lying.